### ISSUE TITLE
Aria-Live attribute not working for screen readers

### ISSUE THREAD

**Issue by amichel86** [nolimits4web/swiper#6221]
Hello, I am working with an accessibility expert, she took a look at Swiper and told me this. Is this something fixable on your end? Thanks!

> The aria-live attribute does not work, because the elements do not disappear and appear. You should know that for the aria-live to announce new content, it must know that there is new content. To do this, the hidden slides should be hidden by display:none and when they appear, we remove the display:none. The aria-live will then announce the new content of the slide.

**Comment by amichel86** [comment]
Also "slideRole" in the ay11 API doesn't change the role on a .swiper-slide, group is always shown.

**Comment by nolimits4web** [comment]
Issue is closed because of outdated/irrelevant/not actual/needed more information/inactivity.

If this issue is still actual and reproducible for latest version of Swiper, please create new issue and fill the issue template correctly:
* Clearly describe the issue including steps to reproduce when it is a bug.
* Make sure you fill in the earliest version that you know has the issue.
* Provide live link or JSFiddle/Codepen or website with issue
